bump-up.command

```diff
@@ -92,6 +92,23 @@ else
   echo "helix-rest/$ivy_file not exist"
 fi
 
+echo "bump up helix-lock/pom.xml"
+sed -i "s/${version}/${new_version}/g" helix-lock/pom.xml
+grep -C 1 "$new_version" helix-lock/pom.xml
+# git diff helix-lock/pom.xml
+
+ivy_file="helix-lock-"$version".ivy"
+new_ivy_file="helix-lock-"$new_version".ivy"
+# echo "$ivy_file"
+if [ -f helix-rest/$ivy_file ]; then
+  echo "bump up helix-lock/$ivy_file"
+  git mv "helix-lock/$ivy_file" "helix-lock/$new_ivy_file"
+  sed -i "s/${version}/${new_version}/g" "helix-lock/$new_ivy_file"
+  grep -C 1 "$new_version" "helix-lock/$new_ivy_file"
+else
+  echo "helix-lock/$ivy_file not exist"
+fi
```

Comment:
Are we planning on adding ivy files? Otherwise, this won't do anything...

Reply:
yeah, ivy files are necessary when we build release internally. Otherwise it won't work.